<launch>
    <node pkg="prometheus_detection" type="web_cam" name="web_cam" output="screen">
        <param name="camera_type" type="int" value="3" />
        <param name="cam_id" type="int" value="0" />
	    <param name="resize_h" type="int" value="360" />
        <param name="resize_w" type="int" value="640" />
    </node>
</launch>
